ntcore/src/main/native/include/networktables/Topic.inc

```diff
@@ -6,18 +6,13 @@
 
 #include <string>
 
-#include "networktables/NetworkTableInstance.h"
 #include "networktables/NetworkTableType.h"
 #include "networktables/Topic.h"
 #include "ntcore_c.h"
 #include "ntcore_cpp.h"
 
 namespace nt {
 
-inline NetworkTableInstance Topic::GetInstance() const {
-  return NetworkTableInstance{GetInstanceFromHandle(m_handle)};
-}
-
```

Comment:
Any particular reason why this was changed to not inline?

Reply:
This was needed to break a circular header dependency on NetworkTableInstance.h (since we need to include Topic.h from NetworkTable.h in order to support the templated GetStructTopic etc).